src/module/vue/sf-locationsheet.vue

```diff
@@ -112,7 +112,7 @@
   padding: 7px;
 }
 .highlighted {
-  background: lightyellow;
+  background: #33999933;
```

Comment:
When I first saw this, I thought "that can't possibly work in the standard black-and-white theme" but actually it kind of does.
![CleanShot 2022-04-24 at 19 05 23](https://user-images.githubusercontent.com/39902/165009037-3f04516d-d7c2-41b0-9dd2-916616a23b78.jpg)
![CleanShot 2022-04-24 at 19 05 37](https://user-images.githubusercontent.com/39902/165009040-a2906325-6f27-4ecd-b9a0-bd0a16c20007.jpg)

Reply:
It actually does look good. Did you add it to the Ironsworn style? Or is vue file used in both?